sys-process/audit/files/rules.d/99-default.rules

```diff
@@ -1,5 +1,2 @@
 # Always report changes to the audit subsystem itself.
--a exclude,never -F msgtype=CONFIG_CHANGE
-
-# Ignore everything else.
--a exclude,always -F msgtype>0
+-a exclude,never -F msgtype=CONFIG_CHANGE
```

Comment:
```suggestion
-a exclude,never -F msgtype=CONFIG_CHANGE

```

A newline here would be good since these files are concatenated and one may want to run `cat rules.d/*`